### PR TITLE
Ease some rules

### PR DESCRIPTION
**Pull request by mabhub**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-config-makina",
-  "version": "0.1.0",
+  "version": "0.3.0",
```
Comment (jrmi) [inline package.json:3]: Not very important but why do you jump from 0.1 to 0.3 ?

Comment (mabhub) [reply package.json:3]: The [current change](359ce534b0c9fe922eacb6cb5d3eab397674c1ec) is from `0.2.0` to `0.3.0`, but the merge request contains also a [previous version bump](74d8cfd47f55e6e5d390152aa813dc33dfc5697f) from `0.1.0` to `0.2.0` so the full changeset of the merge request display the overall version bump.

**Review by mdartic — APPROVED** [review]
Sounds great for me.
I test on MFS project, and each error / warning is in my opinion improving code readability !

File: package.json (hunk 1 above)
Comment (jrmi) [reply package.json:3]: Fine

**Review by jrmi — APPROVED** [review]

**Pull request merged**